src/localization/successor_manager/successor_manager.cpp

```diff
@@ -22,42 +22,27 @@
 **/
 
 #include "successor_manager/successor_manager.h"
+#include "database/idatabase.h"
+#include "glog/logging.h"
```

Comment:
nit: you have the same include below too 😉

Reply:
You are right, thanks! Removed.